test/integration/spec/localtrackpublication.js

```diff
@@ -109,26 +109,44 @@ describe('LocalTrackPublication', function() {
     [thisRoom, ...thoseRooms].forEach(room => room && room.disconnect());
   });
 
-  it('JSDK-2582 trackDisabled event is not fired when participant joins room with already disabled track', async () => {
-    const roomSid = await createRoom(randomName(), defaults.topology);
-    const options = Object.assign({ name: roomSid }, defaults);
-
-    // Alice joins a room
-    const aliceRoom = await connect(getToken('Alice'), Object.assign({ tracks: [] }, options));
+  [true, false].forEach((trackInitiallyEnabled) => {
+    it(`JSDK-2603: ${trackInitiallyEnabled ? 'trackDisabled' : 'trackEnabled'} should fire only on change`, async () => {
```

Comment:
Should it have been: ${trackInitiallyEnabled ? 'trackEnabled' : 'trackDisabled'}?